docs/PID tuning.md

```diff
@@ -1,33 +1,55 @@
+
+**The I term** also multiplies the error with its gain but in addition sums the product over time. This time variant behaviour causes the I term to be more susceptible to overshooting and oscillations. Hence the I term is tuned much lower than the P term in most cases. The gain is that even small errors can lead to great output values after some time making the I term good for slow, precise corrections. I oscillations a generally distinctively slower than P oscillations (low frequency).
+
+**The D term** attempts to increase system stability by monitoring the rate of change in the error. It counteracts with a rate propotional to the speed of the change and its gain. The D term is by far the hardest to tune because it has a delayed effect on the output and potentially overwrites the other terms momentarily. Often to high D gains will make the craft sluggish to respond in combination to weird slow overshooting issues while the I term is fighting the D term at near zero error.
 
 A video on how to recognise and correct different flight problems caused by PID settings is available here:
 
 https://www.youtube.com/watch?v=YNzqTGEl2xQ
 
-Basically, the goal of the PID controller is to bring the craft's rotation rate in all three axes to the rate that
-you're commanding with your sticks. An error is computed which is the difference between your target rotation rate and
-the actual one measured by the gyroscopes, and the controller tries to bring this error to zero.
+## Software filtering
 
-##PIDs
+Every controller has to deal with two problems.
+
+**The first** is noise either caused by "real noise" of the system or by the sensor itself. Either way noise can cause serious issues ranging from reduced flight times regarding hot motors to uncontrolable random shaking of the craft.
+
+**The second** is delay. Meant is the delay between action (controller output) and reaction (sensor input). The more delay the looser the controller is conected to its environment and therefore can not be tuned as aggressively. Imagine a radiator thermostat as a highly delayed controller.
+
+These two evils are best eliminated at the causer. Balance your props and other fast rotating parts. Make shure the axles are strait and the bearings are still good. Dampening the sensors (flightcontrol) might also help. A detailed discussion about vibration cancellation technics [here](http://diydrones.com/forum/topics/vibration-isolation-and-dampening-of-apm-px4-for-version-2-9).
+
+Nevertheless noise is everywhere. Even at a vibration optimised system. To cope with this Cleanflight (1.10.0.0) has a new feature generally referenced to as software filtering. Some sensor hardware comes with its own filtering features (MPU6050 etc.) but all of them have there own drawbacks. See this #1028 Issue for more information. My personal advice to everyone is to use software filtering instead of hardware filtering or even no filtering. Overall flight performance is significally improved.
```

Comment:
"use software filtering instead of hardware filtering": here I first understood you meant physically dampening the vibration but after reading it again, I think you definitely refer to e.g. the MPU6xxx embedded filtering features. Could be maybe made more clear!? I may also be the only one to misunderstand it (English is not my mother tongue).